spec/design/m-skel/ARM/MachineTypes.thy

```diff
@@ -82,7 +82,7 @@ axiomatization
 where
   irq_oracle_max_irq: "\<forall> n. (irq_oracle n) <= ARM.maxIRQ"
 
-text {* The values the timer device will return (how much time passed since last query) *}
+text \<open> The values the timer device will return (how much time passed since last query) \<close>
```

Comment:
minor, but do we want the spaces around `<` and `>` for `text` blocks?

Reply:
I don't have any strong opinions, happy to remove the spaces if you think that would be more consistent.

Reply:
Minor, but I'd be for removing the spaces for more consistency.